.vscode/tasks.json

```diff
@@ -6,7 +6,7 @@
     {
       "label": "Serve",
       "type": "shell",
-      "command": "bundle exec jekyll serve --force_polling --livereload --baseurl=''",
+      "command": "bundle exec jekyll serve --drafts --force_polling --livereload --baseurl=''",
```

Comment:
🔗 http://localhost:4000/docs/dev/internals/contributing/agent-forwarding/

sorry, for delay am also [watching shogun](https://twitter.com/DerekNonGeneric/status/1769537882388041977)